agent/agentserver/server.go

```diff
@@ -14,35 +14,39 @@
 package agentserver
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	_ "net/http/pprof" // Registers /debug/pprof endpoints in http.DefaultServeMux.
 	"os"
-
-	"github.com/pressly/chi"
-	"github.com/uber-go/tally"
+	"strings"
 
 	"github.com/uber/kraken/build-index/tagclient"
 	"github.com/uber/kraken/core"
+	"github.com/uber/kraken/lib/dockerdaemon"
 	"github.com/uber/kraken/lib/middleware"
 	"github.com/uber/kraken/lib/store"
 	"github.com/uber/kraken/lib/torrent/scheduler"
 	"github.com/uber/kraken/utils/handler"
 	"github.com/uber/kraken/utils/httputil"
+
+	"github.com/pressly/chi"
+	"github.com/uber-go/tally"
 )
 
 // Config defines Server configuration.
 type Config struct{}
 
 // Server defines the agent HTTP server.
 type Server struct {
-	config Config
-	stats  tally.Scope
-	cads   *store.CADownloadStore
-	sched  scheduler.ReloadableScheduler
-	tags   tagclient.Client
+	config    Config
+	stats     tally.Scope
+	cads      *store.CADownloadStore
+	sched     scheduler.ReloadableScheduler
+	tags      tagclient.Client
+	dockerCli dockerdaemon.DockerClient
```

Comment:
Nit: just call this docker.

Reply:
docker is referenced for many meanings in this code base,  so I think it's better to be explicit here.